thoth/user_api/api_v1.py

```diff
@@ -801,7 +801,7 @@ def _get_document(adapter_class, analysis_id: str, name_prefix: str = None, name
                     # Can be:
                     #   - return 500 to user as this is our issue
                     raise ValueError(f"Unreachable - unknown workflow state: {status}")
-            except NotFoundException:
+            except (OpenShiftNotFound, NotFoundException):
```

Comment:
You can directly catch `OpenShiftNotFound`. `NotFoundException` is specific to user-api and will never be raised by methods present in the `try` block.

Looking into sources, this can be applied also elsewhere.

Reply:
I will change that. Thanks!